webf/lib/src/html/img.dart

```diff
@@ -374,10 +450,6 @@ class ImageElement extends Element {
   void _stopListeningStream({bool keepStreamAlive = false}) {
     if (!_isListeningStream) return;
 
-    if (keepStreamAlive && _completerHandle == null && _cachedImageStream?.completer != null) {
```

Comment:
Why this line are removed

Reply:
I used another way to keep the image alive.

In the previous, the listener will remove when the render object detach. So must be to keep the image stream alive when it attached back.
I move those to dispose lifecycle, So don't keep the stream alive.